src/components/store/greetingsSlice.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
```

Comment:
- [ ] Please avoid disabling linters as they are very useful to keep a consistent code format and ensure the use of best practices. Instead, please consider solving the errors! 🤓 🔧